fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/fluid/FluidVariantAttributeHandler.java

```diff
@@ -41,7 +45,14 @@ public interface FluidVariantAttributeHandler {
 	 * Return the name that should be used for the passed fluid variant.
 	 */
 	default Text getName(FluidVariant fluidVariant) {
-		return fluidVariant.getFluid().getDefaultState().getBlockState().getBlock().getName();
+		Block fluidBlock = fluidVariant.getFluid().getDefaultState().getBlockState().getBlock();
+
+		if (fluidBlock == Blocks.AIR) {
```

Comment:
This check for `fluidVariant.getFluid() != Fluids.EMPTY` to avoid changing vanilla behaviour as the translation key would be different.